boards/common/atmega/board.c

```diff
@@ -23,10 +23,25 @@
 #include "irq.h"
 #include "periph/gpio.h"
 
+#ifndef CPU_ATMEGA_CLK_SCALE_INIT
+#define CPU_ATMEGA_CLK_SCALE_INIT    CPU_ATMEGA_CLK_SCALE_DIV1
+#endif
+
 void led_init(void);
 
+static inline void set_prescaler(void)
```

Comment:
This function looks CPU-related to me. Would it be problematic to move it to `cpu/atmega_common/include/cpu.h` together with the `#define` you added above? Maybe rename it to `atmega_set_prescaler` then.

Reply:
Note to others: bergzand and I discussed this in IRC. There are some issues with moving it due to wanting to have the ability to override CPU_ATMEGA_CLK_SCALE_INIT in board.h, which would require adding an include for board.h to cpu.h if it is moved. We haven't decided yet the best way to handle this.